tools/freeze-requirements.sh

```diff
@@ -22,12 +22,12 @@ dpkg -x ./*.deb .
 cp -r usr/lib/python3/dist-packages/* "$site_pkgs"
 popd
 
-pip install -e .
-pip freeze --exclude-editable > requirements.txt
+uv pip install -e .
+uv pip freeze --exclude-editable > requirements.txt
```

Comment:
Will it be in future replaced with `pip-compile` / some sort of the lock file and this is only temporary to use pip to generate it instead of `pip`?

Reply:
Eventually we can get rid of this script altogether when we have uv lockfiles.